doc/references.bib

```diff
@@ -2381,3 +2381,25 @@ @article{CroftBarry2000
   pages = {5--19},
   doi = {10.1016/S0987-7053(00)00055-1}
 }
+
+@inproceedings{AdlerEtAl2022,
+  author={Adler, Amir and Wax, Mati and Pantazis, Dimitrios},
+  booktitle={2022 IEEE 19th International Symposium on Biomedical Imaging (ISBI)}, 
+  title={Brain Source Localization by Alternating Projection}, 
+  year={2022},
+  volume={},
+  number={},
+  pages={1-5},
+  doi={10.1109/ISBI52829.2022.9761604}
+}
+
+@misc{AdlerEtAl2019,
+  doi = {10.48550/ARXIV.1908.11416},    
+  author = {Adler, Amir and Wax, Mati and Pantazis, Dimitrios},  
+  keywords = {Signal Processing (eess.SP), FOS: Electrical engineering, electronic engineering, information engineering, FOS: Electrical engineering, electronic engineering, information engineering},  
+  title = {Localization of MEG and EEG Brain Signals by Alternating Projection},  
+  publisher = {arXiv},  
+  year = {2019},
+  url = {https://arxiv.org/abs/1908.11416},
+  copyright = {arXiv.org perpetual, non-exclusive license}
+}
```

Comment:
Please include DOI (preferred) or URL, but not both. The DOI will generate a link, thus providing both results in duplicated links in the documentation.